core/src/read.jl

```diff
@@ -985,10 +1003,10 @@ function user_demand_time!(
         )
         demand[user_demand_idx, priority_idx] = demand_p_itp(0.0)
 
-        if is_valid
+        if is_valid_demand
             demand_itp[user_demand_idx][priority_idx] = demand_p_itp
         else
-            @error "The demand(t) relationship for UserDemand $node_id of priority $p from the time table has repeated timestamps, this can not be interpolated."
+            @error "The demand(t) relationship for UserDemand $(first_row.node_id) of priority $priority_idx from the time table has repeated timestamps, this can not be interpolated."
```

Comment:
```suggestion
            @error "The demand(t) relationship for UserDemand $(first_row.node_id) of priority $(first_row.priority) from the time table has repeated timestamps, this can not be interpolated."
```

`priority_idx` is not exposed to the modeller, this is something that is only used internally, so that provided priorities do not have to be consecutive